mllib/src/main/scala/org/apache/spark/ml/Pipeline.scala

```diff
@@ -212,7 +212,7 @@ object Pipeline extends MLReadable[Pipeline] {
     }
   }
 
-  /** Methods for [[MLReader]] and [[MLWriter]] shared between [[Pipeline]] and [[PipelineModel]] */
+  /** Methods for `MLReader` and `MLWriter` shared between [[Pipeline]] and [[PipelineModel]] */
```

Comment:
Several of these instances are because for whatever reason visibility is wrong in the generated javadoc and so it fails. It seemed more worth it to zap the error than retain these links.